lib/css_parser/parser.rb

```diff
@@ -504,11 +577,11 @@ def read_remote_file(uri) # :nodoc:
 
           res = http.get(uri.request_uri, {'User-Agent' => USER_AGENT, 'Accept-Encoding' => 'gzip'})
           src = res.body
-          charset = fh.respond_to?(:charset) ? fh.charset : 'utf-8'
+          charset = res.respond_to?(:charset) ? res.encoding : 'utf-8'
```

Comment:
we should get rid of this charset detection and drop 1.8 ... but that's for another PR